docs/cookbook/fastapi.md

```diff
@@ -0,0 +1,7 @@
+# FastAPI
+
+!!! note
+
+    Take a look at the [code](https://github.com/Mirascope/mirascope/blob/main/cookbook/api_examples/async_create_example.py) in our repo.
```

Comment:
1. should we rename `api_examples` to `fastapi_examples`
2. should we link the directory here? both files? or just async like you have?

Reply:
1. yes
2. since this is placeholder, doesn't really matter. i think just one will suffice until we upgrade this section